app/views/groupings/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= render 'organizations/organization_banner' %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h2 class="site-content-heading"><%= @grouping.title %></h2>
+  </div>
+
+  <div class="site-content-body">
+    <% if @grouping.groups.present? %>
+      <div class="team-grid">
+        <div class="columns">
+          <div class="one-half column">
+            <% @grouping.groups.each do |group| %>
+                <% if cycle('odd', 'even') == 'odd' %>
+                    <%= render partial: "group", locals: { group: group } %>
+                <% end %>
+            <% end %>
+          </div>
+          <div class="one-half column">
+            <% @grouping.groups.each do |group| %>
+                <% if cycle('odd', 'even') == 'even' %>
+                    <%= render partial: "group", locals: { group: group } %>
+                <% end %>
+            <% end %>
+          </div>
```

Comment:
Visually, I think the Masonry-style layout is nice. A couple thoughts though:

1. There doesn't seem to be any sort of affordance for users to know that these are drag and droppable. If nothing else, there should probably be a one-liner at the top of the page saying so.

2. If you're dragging an avatar to a drop area that's above or below the current viewport, that operation will probably be impractical. Will there be a fallback method?

3. The site uses a gray heading and white content area for the various boxes and these boxes flip that to have a gray content area.

4. Minor design nit: the line with the student count should be tightened up to reduce the whitespace above it.